src/style.css

```diff
@@ -1 +1,82 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+  margin: auto;
+  padding: 0 50px;
+  height: 100%;
+}
+
+.moyo_logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+}
+
+.navigation {
+  display: flex;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
```

Comment:
You should apply `box-shadow` and `background-color` for the whole header, not only for navigation:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/52370890/220964782-6ce93ae6-ffc2-45ba-bf6b-040aadb8294b.png">